org.eclipse.scanning.malcolm.core/src/org/eclipse/scanning/malcolm/core/MalcolmEventDelegate.java

```diff
@@ -15,41 +15,20 @@
 import java.util.Collections;
 import java.util.LinkedHashSet;
 
-import org.eclipse.scanning.api.device.IRunnableDevice;
 import org.eclipse.scanning.api.event.scan.DeviceState;
-import org.eclipse.scanning.api.malcolm.connector.IMalcolmConnectorService;
 import org.eclipse.scanning.api.malcolm.event.IMalcolmListener;
 import org.eclipse.scanning.api.malcolm.event.MalcolmEvent;
 import org.eclipse.scanning.api.malcolm.event.MalcolmEventBean;
-import org.eclipse.scanning.api.malcolm.message.MalcolmMessage;
-import org.eclipse.scanning.api.points.IPosition;
-import org.eclipse.scanning.api.scan.ScanningException;
-import org.eclipse.scanning.api.scan.event.IRunListener;
-import org.eclipse.scanning.api.scan.event.RunEvent;
 
 public class MalcolmEventDelegate {
 
-	private String          topicName;
-
 	// listeners
-	private Collection<IMalcolmListener<MalcolmEventBean>> mlisteners;
+	private Collection<IMalcolmListener<MalcolmEventBean>> listeners;
 
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "topicName" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)